pages/400.html

```diff
@@ -0,0 +1,33 @@
+
+<html>
+    <head>
+        <link type="text/css" rel="stylesheet" media="all" href="error_style/style.css">
+        <link type="text/css" rel="stylesheet" media="all" href="error_style/bootstrap_style.css">
+        <link type="text/css" rel="stylesheet" media="all" href="error_style/additional_style.css">
+    </head>
+    <body>
+        <section class="section section-primary section-lead">
+            <div class="container">
+                <div class="row">
+                    <div class="col-sm-6 flower-container">
+                        <div class="flower-home"></div>
+                    </div>
+                    <div class="col-sm-6 container-welcome">
+                        <div class="welcome-container-inner">
+                            <h1>Your client has submitted an invalid request</h1>
```

Comment:
Are translations possible in this environment? I don't see how, commcare-hq-errorpages pages are static files served outside the django environment